### PR TITLE
Fix translation & html syntax

### PR DESCRIPTION
**Pull request by kosssi**
- https://developers.whatwg.org/grouping-content.html#the-p-element

### CODE REVIEW

File: www-src/app/templates/permissions.jade [hunk 1]
```diff
@@ -3,14 +3,13 @@
...
-                        li
-                            b= key
-                            | :
-                            = t(obj.description)
+            p= t('permissions explanations')
+            ul
+                - each obj, key in permissions
+                    li
+                        b= key
+                        | :&nbsp;
```
Comment (nono) [inline www-src/app/templates/permissions.jade]: @kosssi why a non-breaking space after the `:`?

Comment (kosssi) [inline www-src/app/templates/permissions.jade]: Else we are:like this
                ^ is better with a space

Comment (nono) [inline www-src/app/templates/permissions.jade]: And a normal space does not work?

Comment (kosssi) [inline www-src/app/templates/permissions.jade]: most IDE remove it ;)

Comment (nono) [inline www-src/app/templates/permissions.jade]: OK

**Comment by nono** [comment]
The build failed. I don't think it's related to your change. Do you confirm?

**Comment by kosssi** [comment]
Yes I don't know why...

**Pull request merged**